packages/aws-cdk-lib/aws-appconfig/lib/environment.ts

```diff
@@ -543,4 +559,19 @@ export interface IEnvironment extends IResource {
    * @param extension The extension to create an association for
    */
   addExtension(extension: IExtension): void;
+
+  /**
+   * Grant the given identity the specified actions
+   * @param grantee the identity to be granted the actions
+   * @param actions the data-access actions
+   */
+  grant(grantee: iam.IGrantable, ...actions: string[]): iam.Grant;
+
+  /**
+   * Grants read configuration permissions for this environment
+   * to an IAM principal (Role/Group/User).
```

Comment:
Is the `Grantable` interface === IAM principal? Or is it even broader? 
Are other docstrings for grant methods also describing this in this way?

Reply:
@daschaa 
From my understanding, the Grantable interface === IAM principal.
I have revised the explanation referring to the documentation of the grant method of the DynamoDB Table v2 Construct.

https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_dynamodb.TableV2.html#grantwbrreadwbrwritewbrdatagrantee

Reply:
@mazyu36 Awesome! Thank you very much for the explanation!